app/gui2/src/components/GraphEditor.vue

```diff
@@ -595,17 +592,22 @@ async function handleFileDrop(event: DragEvent) {
   const MULTIPLE_FILES_GAP = 50
 
   if (!event.dataTransfer?.items) return
+  const projectRootId = await projectStore.projectRootId
+  if (projectRootId == null) {
+    console.error('Could not find project root, uploading not possible.')
```

Comment:
I think we should also display a toast, so the user is informed that their file drop hit some issue in our app.